packages/remark-stringify/readme.md

```diff
@@ -207,11 +207,12 @@ Increment ordered list item numbers (`boolean`, default: `true`).
 
 When `false`, all list item numbers will be the same.
 
-###### `options.definitionBlankLine`
+###### `options.tightDefinitions`
 
-Separate definitions with a blank line (`boolean`, default: `true`).
+Separate definitions with only one newline (`boolean`, default: `false`).
```

Comment:
```suggestion
Separate definitions with a single line feed (`boolean`, default: `false`).
```

Reply:
I personally don't use the term “line feed” much (and I haven't seen it used much) except when talking about DOS vs. Unix end-of-line characters or typewriters. I come from the era of C and Unix, which, I believe, is the same source of `'\n'` for “newline.” But you do use it in a number of code variable names and comments, so as long as it's consistent, it's fine with me.